### PR TITLE
Refactored code

### PR DESCRIPTION
**Pull request by BayLadyCoder**

### CODE REVIEW

File: src/lib/formValidator.js [hunk 1]
```diff
@@ -1,72 +1,97 @@
...
+
+  if (!host) {
     isError = true;
     err.host_error = 'Required';
   }
 
-  if (values.degree || values.degree !== undefined) {
-    if (!values.degree) {
+  if (type === 'Education') {
+    if (!degree) {
```
Comment (dfkoh) [inline src/lib/formValidator.js]: Use `&&` here

File: src/lib/formValidator.js [hunk 2]
```diff
@@ -1,72 +1,97 @@
...
+    if (!location) {
       isError = true;
       err.location_error = 'Required';
     }
   }
-  // if (!values.end_month) {
-  if (type !== 'Accomplishment' && values.is_current === false) {
-    if (!values.end_month || !monthFullNames.includes(values.end_month)) {
+
+  // require end_mont and end_year if is_current is false
```
Comment (dfkoh) [inline src/lib/formValidator.js]: `end_month`

File: src/lib/formValidator.js (hunk 1 above)
Comment (BayLadyCoder) [reply src/lib/formValidator.js]: done

File: src/lib/formValidator.js (hunk 2 above)
Comment (BayLadyCoder) [reply src/lib/formValidator.js]: thanks

**Pull request merged**